components/Buttons/CommunityButton.js

```diff
@@ -5,17 +5,15 @@ import { style } from './CommunityButtonStyle'
 
 export default class CommunityButton extends Component {
 
-  communityFeedback (communityMode) {
-    return communityMode
-    ? 'Sharing location with community'
-    : 'Stealth mode – not socializing'
-  }
-
   render () {
+		const title = this.props.communityMode
```

Comment:
[unindent](https://www.youtube.com/watch?v=Zf8ir_gvbwA) 🎶

Reply:
is it on my Atom (even on VScode) ... any clues ??